src/cljc/matcher_combinators/standalone.cljc

```diff
@@ -3,14 +3,37 @@
             [matcher-combinators.core :as core]
             [matcher-combinators.parser]))
 
+(defn match
+  "Returns a map indicating whether the `actual` value matches the `matcher`.
+
+  `matcher` can be a matcher-combinators matcher, a predicate function of
+  actual, an expression that returns a value, or a literal value.
+
+  `actual` can be an expression that returns the actual value, or a literal.
+
+  Return map includes the following keys:
+
+  - :matcher-combinators.result/type
+    - either :match or :mismatch
+  - :matcher-combinators.result/value
+    - when :matcher-combinators.result/type is :match, binds to the actual value,
+      else a map indicating the expected and actual values "
+  [matcher actual]
+  (core/match matcher actual))
+
 (s/fdef match?
   :args (s/alt :partial (s/cat :matcher (partial satisfies? core/Matcher))
                :full    (s/cat :matcher (partial satisfies? core/Matcher)
                                :actual any?))
   :ret boolean?)
 
 (defn match?
-  "Does the value match the provided matcher-combinator?"
+  "Given a `matcher` and `actual`, returns `true` if
+  :matcher-combinators.result/type in the map returned
+  by `(match matcher actual)` is :match. Else, returns `false.`
```

Comment:
`:matcher-combinators.result/type` is an implementation detail and I don't know if it is helpful to talk about it here